apps/api/src/sibyl/server.py

```
@@ -787,8 +810,8 @@ async def _add_mcp_entity(
     )
     write_decision = _authorize_mcp_memory_write(
         ctx=ctx,
-        memory_scope="project" if project else "private",
-        scope_key=project,
+        memory_scope=memory_scope,
```

Ruff (0.15.12)

[error] Undefined name `memory_scope`

(F821)